__tests__/app.test.js

```diff
@@ -154,3 +154,91 @@ test("400: will respond with a 400 when the article_id is invalid", () => {
       expect(body.msg).toBe("Bad Request");
     });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("POST 201: accepts a request of an object with username and body property, and responds with the posted comment object", () => {
+    const requestBody = {
+      username: "lurker",
+      body: "I am sure, and don't call me Shirley.",
+    };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(requestBody)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.result).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: 0,
+          created_at: expect.any(String),
+          author: "lurker",
+          body: "I am sure, and don't call me Shirley.",
+          article_id: 5,
+        });
+      });
+  });
+});
+test("POST 400: responds with a 400 status code and error message if user inputs an invalid article ID", () => {
+  const requestBody = {
+    username: "lurker",
+    body: "I am sure, and don't call me Shirley.",
+  };
+  return request(app)
+    .post("/api/articles/orange/comments")
+    .send(requestBody)
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad Request");
+    });
+});
```

Comment:
Really nice test but dont forget we also need to ensure we give the right response when given an article id that is valid but doesnt exist. I.e. /api/articles/999/comments